vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ElementSteps.java

```diff
@@ -216,21 +222,63 @@ public void doesElementHaveRightCss(String cssName, String cssValue)
     }
 
     /**
-     * Checks that the context <b>element</b> has an expected <b>CSS property</b>
+     * Checks that the context <b>element</b> has an expected <b>CSS property</b>.
+     * <p>
+     * If the comparison rule is <b>{@code IS_EQUAL_TO}</b> and both the actual and expected CSS values 
+     * are colors in RGB or RGBA format (with an <b>alpha channel of 1</b> in RGBA [{@code rgba(r,g,b,1)}]),
+     * then both values are normalized to the RGB [{@code rgb(r,g,b)}] format before the comparison. 
+     * <p>
+     * Normalization is <b>not performed</b> for other comparison rules or color representations.
      * @param cssName A name of the <b>CSS property</b>
-     * @param comparisonRule is equal to, contains, does not contain
+     * @param comparisonRule is equal to, contains, does not contain, matches
      * @param cssValue An expected value of <b>CSS property</b>
      */
     @Then("context element has CSS property `$cssName` with value that $comparisonRule `$cssValue`")
     public void doesElementHaveRightCss(String cssName, StringComparisonRule comparisonRule, String cssValue)
     {
         uiContext.getSearchContext(WebElement.class).ifPresent(element -> {
-            Matcher<String> matcher = comparisonRule.createMatcher(cssValue);
             String actualCssValue = webElementActions.getCssValue(element, cssName);
-            descriptiveSoftAssert.assertThat("Element css property value is", actualCssValue, matcher);
```

Comment:
we can't change the assertion message (it's a breaking change), please use new assertion message only for rgb vs rgba comparison